openedx/core/djangoapps/discussions/utils.py

```diff
@@ -191,3 +193,22 @@ def get_course_division_scheme(course_discussion_settings: CourseDiscussionSetti
     ):
         division_scheme = CourseDiscussionSettings.NONE
     return division_scheme
+
+
+def use_discussions_mfe(org) -> bool:
+    """
+    Checks with the org if the tenant enables the
+    Discussions MFE.
+    Returns:
+        True if the MFE setting is activated, by default
+        the MFE is deactivated
+    """
+    ENABLE_MFE_FOR_TESTING = SettingDictToggle(
+        "FEATURES", "ENABLE_MFE_FOR_TESTING", default=False, module_name=__name__
+    ).is_enabled()
+
+    use_discussions = configuration_helpers.get_value_for_org(
+        org, "USE_DISCUSSIONS_MFE_FRONTEND", ENABLE_MFE_FOR_TESTING or False
```

Comment:
I agree with the Dianis suggestion:

```suggestion
        org, "USE_DISCUSSIONS_MFE", ENABLE_MFE_FOR_TESTING or False
```